src/SonarQube.Client/Api/V9_7/SearcHotspotRequest.cs

```diff
@@ -33,7 +33,7 @@ namespace SonarQube.Client.Api.V9_7
     public class SearchHotspotRequest : PagedRequestBase<SonarQubeHotspotSearch>, ISearchHotspotRequest
     {
         [JsonProperty("projectKey")]
```

Comment:
This should be renaming the JSON property from `projectKey` to `project`, shouldn't it?

https://next.sonarqube.com/sonarqube/web_api/api/hotspots?query=hotspot